docs/config-example.md

```diff
@@ -80,15 +80,15 @@ spec:
       ipipMode: Always  # IPIP Mode to use for the IPv4 POOL created at start up. If set to a value other than Never, vxlanMode should be set to "Never". [Always | CrossSubnet | Never] [Default: Always]
       vxlanMode: Never  # VXLAN Mode to use for the IPv4 POOL created at start up. If set to a value other than Never, ipipMode should be set to "Never". [Always | CrossSubnet | Never] [Default: Never]
       vethMTU: 0  # The maximum transmission unit (MTU) setting determines the largest packet size that can be transmitted through your network. By default, MTU is auto-detected. [Default: 0]
-    kubePodsCIDR: 10.233.64.0/18
-    kubeServiceCIDR: 10.233.0.0/18
+    kubePodsCIDR: 10.233.64.0/18,fc00::/48
```

Comment:
why user want to support ipv6, but not set ipv6 pods and service cird ? so if user relly want to support ipv6, the config must set with node ipv6 addresses and (pod,service)cidr.